wordpress-mobile/WordPress-iOS
# Fix an issue where the Login navigation bar disappears during a pop a…

Fixes #19437

## Related PR
This PR depends on the following:
- https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/687

## Issues
- When swiping back to Login Prologue screen ( for both WordPress and Jetpack ), the navigation bar becomes invisible during the pop animation ( see WordPress screen recordings )
- When tapping "Enter your existing site address", the navigation bar changes its appearance before the push animation completes ( see Jetpack screenshots ).

|             | Before | After |
| ------ | ------ | ----- |
| **WordPress** | ![](https://user-images.githubusercontent.com/9609223/195313350-e591d375-3e94-4780-b994-62b94156308b.MP4) |  ![](https://user-images.githubusercontent.com/9609223/195615159-d6b57aed-c68f-4888-99bd-946b04183754.mp4) |
| **Jetpack** | ![jp-before](https://user-images.githubusercontent.com/9609223/195619805-5ff71e2a-afa5-43d2-99db-16cc38af8235.png) | ![jp-after](https://user-images.githubusercontent.com/9609223/195619839-caf8a904-497d-4cf4-a90b-8a3e07ab79bc.png) |
 
## Test Instructions
Repeat the following steps for both WordPress and Jetpack apps.
1. Open the app, and log out of existing sessions to reach the Landing Screen.
2. Tap "Continue with WordPress.com".
3. Go back to previous screen, either by tapping "Back" button or using the swipe back gesture.
4. **Expect:** Navigation bar shouldn't disappear during the transition.
5. Tap "Enter your existing site address".
6. **Expect:** Navigation should change its appearance alongside the push transition.
7. Go back to previous screen, either by tapping "Back" button or using the swipe back gesture.
8. **Expect:** Navigation bar shouldn't disappear during the transition.

## Regression Notes
1. Potential unintended areas of impact
None.

3. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests.

5. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:
